pkgs/development/python-modules/netaddr/default.nix

```diff
@@ -4,33 +4,22 @@
 , pytest
 , fetchpatch
 , glibcLocales
+, pythonOlder
+, importlib-resources
 }:
 
 buildPythonPackage rec {
   pname = "netaddr";
-  version = "0.7.19";
+  version = "0.8.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "38aeec7cdd035081d3a4c306394b19d677623bf76fa0913f6695127c7753aefd";
+    sha256 = "0hx2npi0wnhwlcybilgwlddw6qffx1mb7a3sj4p9s7bvl33mgk6n";
   };
 
   LC_ALL = "en_US.UTF-8";
-  checkInputs = [ glibcLocales pytest ];
-
-  checkPhase = ''
-    # fails on python3.7: https://github.com/drkjam/netaddr/issues/182
-    py.test \
-      -k 'not test_ip_splitter_remove_prefix_larger_than_input_range' \
-      netaddr/tests
-  '';
-
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/drkjam/netaddr/commit/2ab73f10be7069c9412e853d2d0caf29bd624012.patch";
-      sha256 = "0s1cdn9v5alpviabhcjmzc0m2pnpq9dh2fnnk2x96dnry1pshg39";
-    })
-  ];
+  checkInputs = [ glibcLocales pytest ]
+    ++ stdenv.lib.optionals (pythonOlder "3.7") [ importlib-resources ] ;
```

Comment:
I think this should be a ``propagatedBuildInput`` b/c It's used in the main package.
https://github.com/netaddr/netaddr/blob/ebbaf8b30ef6b6c2a2845bb1cf99f334db6d8700/netaddr/compat.py